PodcastGenerator/admin/pg_integrations.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Right now I think it's overkill to have an own file for this. Could you just place this into the generic PG settings? Maybe we can go back to this approach once we got more integrations

Reply:
pg_integrations.php exists because I also have an integration for Podcast Index which I haven't PR'd yet since it relies on use of Composer. Unfortunately unlike getid3, the podcastindex package has further Composer dependencies so it's beyond the point where we could simply add it into the repo or Git submodule it.

For now I'll move the WebSub setting into pg_config.php but that change will need to be rolled back if/when the Podcast Index pinging comes into play.